src/aaz_dev/command/controller/command_tree.py

```diff
@@ -0,0 +1,647 @@
+import json
+import logging
+import os
+import re
+
+from command.model.configuration import CMDHelp, CMDCommandExample
+from command.model.specs import CMDSpecsCommandGroup, CMDSpecsCommand, CMDSpecsResource, CMDSpecsCommandVersion, \
+    CMDSpecsCommandTree
+from command.model.specs._command_tree import CMDSpecsSimpleCommand, CMDSpecsSimpleCommandGroup, \
+    CMDSpecsSimpleCommandTree
+from utils import exceptions
+
+logger = logging.getLogger(__name__)
+
+
+def _build_simple_command(names):
+    # uri = '/Commands/' + '/'.join(names[:-1]) + f'/_{names[-1]}.md'
+    command = CMDSpecsSimpleCommand()
+    command.names = names
+    return command
+
+
+def _build_simple_command_group(names, aaz_path):
+    """
+    Build Simple Command Group from directory
+    """
+    rel_names = names
+    if len(names) == 1 and names[0] == 'aaz':
+        rel_names = []
+    # uri = '/Commands/' + '/'.join(rel_names) + f'/readme.md'
+    full_path = os.path.join(aaz_path, 'Commands', *rel_names)
+    if rel_names and not os.path.exists(os.path.join(full_path, 'readme.md')):
+        return None
+    commands = {}
+    command_groups = {}
+    for dir in os.listdir(full_path):
+        if os.path.isfile(os.path.join(full_path, dir)):
+            if dir == 'readme.md' or dir == 'tree.json':
+                continue
```

Comment:
It's better to ignore all not markdown files
```suggestion
            if not dir.endswith('.md') or dir == 'readme.md':
                continue
```